templates/docs/examples/patterns/search-and-filter/with-chips.html

```diff
@@ -0,0 +1,121 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Search and filter / Default{% endblock %}
```

Comment:
You have both examples with same title. 
```suggestion
{% block title %}Search and filter / With chips{% endblock %}
```

Reply:
Thanks done